test/csharp/testing-matchers/test.ts

```diff
@@ -0,0 +1,21 @@
+import { TestDriver } from "../../test-helper";
+
+describe("csharp testing assertions", () => {
+  let driver: TestDriver;
+
+  beforeAll(async () => {
+    driver = new TestDriver(__dirname);
+    await driver.setupCsharpProject();
+    await driver.copyFiles("Test.cs");
+    await driver.copyFiles("update-csproj.sh");
+    await driver.exec("dotnet add package HashiCorp.Cdktf.Providers.Aws");
+    await driver.exec("dotnet add package Microsoft.NET.Test.Sdk");
+    await driver.exec("dotnet add package xunit");
+    await driver.exec("dotnet add package xunit.runner.visualstudio");
+    await driver.exec("bash update-csproj.sh");
+  });
+
+  test("run testing suite", async () => {
+    await driver.exec("dotnet test");
```

Comment:
I would put the result in a var (contains stdout and stderr) and check if there is a tests passed successfully output